docs/what-can-be-faked.md

```diff
@@ -43,4 +44,4 @@ Note that this means that static members, including extension methods,
 
 ### Methods that return values by reference
 
-Methods that return values by reference (officially called "[reference return values](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/ref-returns#what-is-a-reference-return-value)") cannot be invoked on a Fake. Any attempt to do so will result in a `NullReferenceException` being thrown.
\ No newline at end of file
+Methods that return values by reference (officially called "[reference return values](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/ref-returns#what-is-a-reference-return-value)") cannot be invoked on a Fake. Any attempt to do so will result in a `NullReferenceException` being thrown.

```

Comment:
I don't understand why this line is shown as a difference. It's the same on both sides 🤔

Reply:
Turn on whitespace differences. My editor added a new line. I can expunge if you like.

Reply:
No, don't bother! Thanks